src/utils/semver.js

```diff
@@ -6,6 +10,9 @@ import { getLocalRegistryURL } from "@nodesecure/npm-registry-sdk";
 // Import Internal Dependencies
 import { NPM_TOKEN } from "./index.js";
 
+// CONSTANTS
+const KVersionsToWarn = ["0.0.0", "0.0", "0"];
```

Comment:
```suggestion
const kVersionsToWarn = ["0.0.0", "0.0", "0"];
```

I would use a `Set` but i guess an array is ok
```suggestion
const kVersionsToWarn = new Set(["0.0.0", "0.0", "0"]);
```